.circleci/config.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           command: |
             set -euo pipefail
             export CELO_MONOREPO_DIR="$HOME/geth/celo-monorepo"
-            git clone --depth 1 https://github.com/celo-org/celo-monorepo.git ${CELO_MONOREPO_DIR} -b master
+            git clone --depth 1 https://github.com/celo-org/celo-monorepo.git ${CELO_MONOREPO_DIR} -b m-chrzan/fixidity
```

Comment:
You only need to repoint the e2e tests for the *first* PR that you merge, and you can leave the other pointing to master.

Reply:
Still suggest this